runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/catalog/TestCatalogHandler.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.engine.test.internal.catalog;
+
+import io.aklivity.zilla.runtime.engine.catalog.CatalogHandler;
+import io.aklivity.zilla.runtime.engine.config.CatalogConfig;
+
+public class TestCatalogHandler implements CatalogHandler
+{
+    public TestCatalogHandler(
+        CatalogConfig schema)
+    {
+    }
+
+    @Override
+    public int register(
+        String schemaDefinition,
+        String subject)
+    {
+        return 1;
+    }
+
+    @Override
+    public String resolve(
+        String subject,
+        String version)
+    {
+        return null;
+    }
+
+    @Override
+    public String resolve(
+        String schemaId)
```

Comment:
Shouldn't schemaId be `int`, also for the return value from `resolve(subject, version)` above?